crypto/chacha/asm/chacha-riscv64-zbb-v.pl

```diff
@@ -305,7 +345,8 @@ sub chacha_quad_round_group {
       $STATE0, $STATE4, $STATE8, $STATE12,
       $STATE1, $STATE5, $STATE9, $STATE13,
       $STATE2, $STATE6, $STATE10, $STATE14,
-      $STATE3, $STATE7, $STATE11, $STATE15]}
+      $STATE3, $STATE7, $STATE11, $STATE15,
+      $V24, $V25, $V26, $V27]}
```

Comment:
just for readability, the comment above `vlsseg_nf_e32_v 8, $V16, $INPUT, $T0` could be `# load the top-half of input data into $V16~$V23` and the comment above `@{[vlsseg_nf_e32_v 8, $V24, $T1, $T0]}` could be `# now load the bottom-half of input data into $V24~$V31` so that readers can more easily infer that `$V24~$V27` at this point are available and can be used as temporary register. A comment above `chacha_quad_round_group` saying `# $V24~$V27 are available now` could be added.

Or more informatively, comments below `map( "v$_", ( 0 .. 31 ) );` could be the usage of each vreg, e.g. `V0~V15 for state, V16~V23 for top-half of input and V24~V31 for bottom-half of input. The vector element of each vreg of the same index, together form the whole state and input (note that vector element width is 32). V24~V27 are available for temporary register at some point. In scalar_data_loop they are used in another way...`

Reply:
Thanks. I added another commit to add these comments, as you suggested.